welder-web.spec

```diff
@@ -0,0 +1,25 @@
+Name: welder-web
+Version: 0
+Release: 0
```

Comment:
It is possible to set the version of package.json with `npm version 1.1.1` but I'm not sure how this would help with importing it into the spec file.